_includes/styles.html

```diff
@@ -5,4 +5,4 @@
 page.styles | uniq %} {% for _list in _styles %}{% for _style in _list %}
 <link rel="stylesheet" href="{{ _style.href | default: _style | relative_url }}" media="{{ _style.media | default: 'screen' }}" />
 {% endfor %}{% endfor %}
-<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous" />
\ No newline at end of file
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous" />
```

Comment:
Created issue to remove this in https://github.com/GSA/digitalgov-pra/issues/328

Reply:
Noted